autoapi/mappers/python.py

```diff
@@ -240,7 +244,7 @@ class PythonModule(TopLevelPythonPythonMapper):
 
     def _resolve_name(self):
         name = self.obj['relative_path']
-        name = name.replace('/', '.')
+        name = name.replace(os.sep, '.')
```

Comment:
This is needed to make it work on Windows - `relative_path` can contain multiple components and without this, the replacement doesn't occur and causes tests to fail.